.github/actions/build/action.yml

```diff
@@ -8,6 +8,12 @@ inputs:
 runs:
   using: composite
   steps:
+    - name: Install dependencies
+      run: npm install
+      shell: bash
```

Comment:
there is a separate action for npm install here -.github/actions/npm_install/action.yml. Can you make the changes in that file if needed?